### PR TITLE
Allow control of multiple instances via dbus

### PR DESCRIPTION
**Pull request by longagofaraway**
## Description
Added another argument to OMXPlayer constructor - dbus_name. Using this name we'll be able to communicate with multiple running instances of omxplayer

### CODE REVIEW

**Comment by willprice** [comment]
Hey, thanks for the pull req. Please update it so that the tests also reflect the new API.

**Comment by longagofaraway** [comment]
Updated.

File: README.md [hunk 1]
```diff
@@ -45,11 +45,9 @@ file_path_or_url = 'path/to/file.mp4'
 # process of fixing this though.
 player = OMXPlayer(file_path_or_url)
 
-# The player will initially be paused
-
-player.play()
-sleep(5)
 player.pause()
```
Comment (willprice) [inline README.md]: Why have you swapped round these lines? Does the player no longer pause initially? If so, that's a bug, I don't want to change the default behaviour thus breaking backwards compatibility.

Comment (longagofaraway) [inline README.md]: The player don't pause initially and it's not my changes. I just fixed the readme.

Comment (longagofaraway) [inline README.md]: ece310aefb41bdc7a8b8a8448c5da19ba00f3938

Comment (willprice) [inline README.md]: @longagofaraway ah, thanks for the commit reference, I was mis-remembering the default.

Going to review and probably merge tonight :)

**Pull request merged**